code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -349,18 +356,19 @@
 	color = "#FFFFFF" // rgb: 255,255,255
 	taste_description = "salt"
 
-/datum/reagent/consumable/sodiumchloride/expose_mob(mob/living/M, methods=TOUCH, reac_volume)
-	if(!istype(M))
+/datum/reagent/consumable/sodiumchloride/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message=TRUE, touch_protection=0)
+	if(!istype(exposed_mob))
 		return
-	if(M.has_bane(BANE_SALT))
-		M.mind.disrupt_spells(-200)
+	SEND_SIGNAL(src, COMSIG_REAGENT_EXPOSE_MOB, exposed_mob, methods, reac_volume, show_message, touch_protection)
```

Comment:
This should be done at the base of the proc.

Reply:
Done